src/main/java/bio/terra/cli/businessobject/SpendProfileUser.java

```diff
@@ -0,0 +1,120 @@
+import bio.terra.cli.service.SpendProfileManagerService.SpendProfilePolicy;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.broadinstitute.dsde.workbench.client.sam.model.AccessPolicyResponseEntry;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Internal representation of a spend profile user (i.e. someone who has permission to spend money
+ * in Terra). This is different from a regular {@link User} because they are never logged in. This
+ * is just a reference to another Terra user who has some permissions on a spend profile. This class
+ * is not part of the current context or state.
+ */
+public class SpendProfileUser {
+  private static final Logger logger = LoggerFactory.getLogger(SpendProfileUser.class);
+
+  private String email;
+  private List<SpendProfilePolicy> policies;
+
+  private SpendProfileUser(String email, List<SpendProfilePolicy> policies) {
+    this.email = email;
+    this.policies = policies;
+  }
+
+  /**
+   * Enable a user on the default WSM spend profile.
+   *
+   * @param email email of the user to add
+   * @param policy policy to assign the user
+   */
+  public static SpendProfileUser enable(String email, SpendProfilePolicy policy) {
+    // call SAM to add a policy + email to the WSM default spend profile resource
+    new SpendProfileManagerService().enableUserForDefaultSpendProfile(policy, email);
+    logger.info("Enabled user on spend profile: email={}, policy={}", email, policy);
+
+    // return a SpendProfileUser = email + all policies (not just the one that was added here)
+    return getUser(email);
+  }
+
+  /**
+   * Disable a user on the default WSM spend profile.
+   *
+   * @param email email of the user to remove
+   * @param policy policy to remove from the user
+   */
+  public static SpendProfileUser disable(String email, SpendProfilePolicy policy) {
+    // call SAM to remove a policy + email from the WSM default spend profile resource
+    new SpendProfileManagerService().disableUserForDefaultSpendProfile(policy, email);
+    logger.info("Disabled user on spend profile: email={}, policy={}", email, policy);
+
+    // return a SpendProfileUser = email + all policies (not just the one that was removed here)
+    return getUser(email);
+  }
+
+  /** Get the spend profile user object. */
+  private static SpendProfileUser getUser(String email) {
+    // lowercase the email so there is a consistent way of looking up the email address
+    // the email address casing in SAM may not match the case of what is provided by the user
+    return listUsersInMap().get(email.toLowerCase());
+  }
+
+  /**
+   * List the users of the default WSM spend profile.
+   *
+   * @return a list of spend profile users
+   */
+  public static List<SpendProfileUser> list() {
+    return listUsersInMap().values().stream().collect(Collectors.toList());
```

Comment:
alphabetize, unless that happens later

Reply:
Yes, the alphabetizing happens in the command classes, because it's just for user-facing output. No need to do it in the internal business logic classes.